vita3k/gui/src/app_selector.cpp

```diff
@@ -304,7 +304,7 @@ void draw_app_selector(GuiState &gui, HostState &host) {
             ImGui::SetColumnWidth(2, COLUMN_SIZE);
             ImGui::SetColumnWidth(3, COLUMN_SIZE);
         }
-        ImGui::SetWindowFontScale(!gui.live_area_font_data.empty() ? 0.82f * scal.x: 1.f);
+        ImGui::SetWindowFontScale(!gui.live_area_font_data.empty() ? 0.82f * scal.x : 1.f);
         ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT);
```

Comment:
Not sure if this is the pr for this...

Reply:
? you have forget, obligatory need run format for pr now :)

Reply:
Well frick lol